stl/inc/chrono

```diff
@@ -5935,7 +5948,7 @@ namespace chrono {
             if (_Specs._Chrono_specs_list.empty()) {
                 _Stream << _Val; // N4885 [time.format]/6
             } else {
-                _Stream.imbue(_FormatCtx.locale());
+                _Stream.imbue(_Specs._Localized ? _FormatCtx.locale() : std::locale::classic());

```

Comment:
Since we're within `std::chrono`, qualifying `locale` is unnecessary.

(In general, `std::` should never appear in the STL's product code - when needed, it should be `_STD` which expands to `::std::` to avoid any ambiguity.)

Reply:
Can't get used to changing stuff inside std =). Dropped redundant qualification.